attributes/default.rb

```diff
@@ -1,5 +1,11 @@
 
 # Override cookbook attributes to install Java 7
-override['java']['openjdk_packages'] = [
-  "openjdk-7-jdk", "openjdk-7-jre-headless"
-]
+#override['java']['openjdk_packages'] = [
+#  "openjdk-7-jdk", "openjdk-7-jre-headless"
+#]
```

Comment:
I'm doing some work with Logstash and Elasticsearch, both of which lean on the community Java cookbook. I have used this to ensure OpenJDK and version 7:

`default['java']['install_flavor'] = 'openjdk'`
`default['java']['jdk_version'] = '7'`